test/ja/ja_casual.test.ts

```diff
@@ -2,27 +2,90 @@ import * as chrono from "../../src/";
 import { testSingleCase } from "../test_util";
 
 test("Test - Single Expression", function () {
-    testSingleCase(chrono.ja, "今日感じたことを忘れずに", new Date(2012, 8 - 1, 10, 12), (result) => {
-        expect(result.index).toBe(0);
-        expect(result.text).toBe("今日");
+    function testToday(pattern: string) {
+        testSingleCase(chrono.ja, `${pattern}感じたことを忘れずに`, new Date(2012, 8 - 1, 10, 12), (result) => {
+            expect(result.index).toBe(0);
+            expect(result.text).toBe(pattern);
+            expect(result.start).not.toBeNull();
+            expect(result.start.get("year")).toBe(2012);
+            expect(result.start.get("month")).toBe(8);
+            expect(result.start.get("day")).toBe(10);
+            expect(result.start).toBeDate(new Date(2012, 8 - 1, 10, 12));
```

Comment:
I think, previously, we test too many things. e.g. only `expect(result.start).toBeDate()` and maybe `expect(result.text)` is enough.

Reply:
Will do so accordingly 👍